dashboards/postgres/main.json

```
@@ -0,0 +1,400 @@
+{
+  "metric_keys": [
+    "vector_pg_up"
+  ],
+  "dashboard": {
+    "title": "PostgreSQL",
+    "description": "",
+    "visuals": [
+      {
+        "title": "pg_stat_database_deadlocks_total",
+        "description": "Number of deadlocks detected in this database.",
+        "line_label": "%name% %db%-%endpoint%-%host%",
```

Speaking of UI limitations, I'd drop `%name%` (the metric name) from here entirely. It's so long that it will push the interesting values (DB, endpoint and host) away from view. (The metric name is already present on the title/description, and there's always the "edit graph" button)

I also tweaked the formatting a bit for readability. I'm not sure what `%endpoint%` is, I've left it out in the suggestion, but do put it back in if you think it'd be meaningful to most users.

```suggestion
        "line_label": "%db% (%host%)",
```

(Same thing would apply for other metrics if you agree)